webapp/src/ts/modals/edit-user/confirm-password-updated.component.html

```diff
@@ -0,0 +1,9 @@
+<mm-modal
+  danger="true"
```

Comment:
```suggestion
```

Remove the "danger" style due that the dialog is only informative → confirmation button is blue instead of red.

Reply:
Hi @mrsarm! I implemented the fix, thanks.
I have one question: in case users click outside the modal, the modal is closed and the login is not triggered. I was trying to disable the closure of the modal in case of user clicking outside the modal in order to force users to clicking on the ok button and overcome this issue, but I have not succeeded. Do you think it is worth adding this feature? If yes, do you know how to implement this. Thanks

Reply:
Good point! Don't worry, I think we just need to pass the `close()` method to a property in the modal definition, tomorrow when working with my laptop I'll point out an example.

Reply:
Fixed in a commit I added to see also whether pushing the change myself would trigger the CI process that failed last time.

Reply:
The issue from CI persists after last push and even re-running it manually, I'm trying to solve it from other channels at Medic so we can merge the PR soon.

Reply:
@mrsarm Thank you very much for the suggestion! Much appreciated. I just made a small change calling directly the private method `windowReload` instead of reloading with `window.location.reload(true)`

Reply:
@dacar1988 , thanks for your patience, as I mention last week we couldn't merge your PR yet because we implemented a new CI pipeline that replaces Travis with Github Actions, and the new implementation didn't allow PRs from external contributors to be evaluated (https://github.com/medic/angular10-migration/issues/131).

We have pushed a fix in the same upstream branch `4750-angular-10-take-2` used in this PR, so your contribution can be used also to test the CI fix. Could you please rebase again your branch with the latest changes in the base branch from our repo and force push the result? that should unlock the CI process that was not working in this PR and let us: :white_check_mark: merge your PR :white_check_mark: verify the CI fix works.

Reply:
Hi! I answered all the comments and pushed the new code.